lib/network/modules/InteractionHandler.js

```diff
@@ -165,8 +165,11 @@ class InteractionHandler {
 
   onContext(event) {
```

Comment:
OK clear. For completeness, I mention that `getNodeAt()` would have worked in the callback handler, as the documentation says.